services/blob/service/src/main/java/org/collectionspace/services/blob/BlobResource.java

```diff
@@ -256,40 +293,40 @@ public Response create(
     @Override
     protected CacheControl getCacheControl(ServiceContext<PoxPayloadIn, PoxPayloadOut> ctx, String cacheKey) {
     	CacheControl result = null;
-    	
+
     	if (cacheKey.matches(DERIVATIVES_REGEX)) {
     		Pattern p = Pattern.compile(DERIVATIVES_REGEX);
     		Matcher m = p.matcher(cacheKey);
     		if (m.find()) {
     			cacheKey = String.format("%s%s*%s", m.group(1), m.group(2), m.group(4));  // Converts something like this "GET/blobs/*/derivatives/Medium/content" into this "GET/blobs/*/derivatives/*/content"
     		}
     	}
-    	
+
     	result = super.getCacheControl(ctx, cacheKey);
     	if (result == null) {
     		result = new CacheControl();
     		result.setMaxAge(DEFAULT_MAX_CACHE_AGE);
     	}
-    	
+
     	return result;
     }
-    
+
     @GET
     @Path("{csid}/content")
     public Response getBlobContent(
     		@PathParam("csid") String csid,
     		@Context Request jaxRsRequest,
     		@Context UriInfo uriInfo) {
     	Response result = null;
-    	ServiceContext<PoxPayloadIn, PoxPayloadOut> ctx = null; 
-    	
+    	ServiceContext<PoxPayloadIn, PoxPayloadOut> ctx = null;
+
     	try {
 	    	ctx = createServiceContext(jaxRsRequest, uriInfo);
 			BlobsCommon blobsCommon = getBlobsCommon(csid);
 	    	StringBuffer mimeType = new StringBuffer();
-	    	InputStream contentStream = getBlobContent(ctx, csid, null /*derivative term*/, mimeType /*will get set*/);
-		    
-	    	Response.ResponseBuilder responseBuilder = Response.ok(contentStream, mimeType.toString());
+			File contentFile = getBlobContentFile(ctx, csid, null /*derivative term*/, mimeType /*will get set*/);
```

Comment:
What's the motivation for this change?

Reply:
This was to allow partial content requests --which are handled by the RESTEasy/JAX-RS framework when given a File instance.  Was not working when given an InputStream instance.